third_party/library.sh

```diff
@@ -115,7 +115,16 @@ metadata:
 ---
 EOF
 
-    ${ISTIO_DIR}/bin/istioctl manifest generate -f "$file"  "$@" | add_crd_label >> "${tmpfile}"
+    if [[ "$file" == *"ambient"* ]] ; then
+      # The options are duplicated but necessary both istio.yaml and command option.
+      "${ISTIO_DIR}"/bin/istioctl manifest \
+           --set profile=ambient \
+	   --set components.ingressGateways[0].enabled=true \
+	   --set components.ingressGateways[0].name=istio-ingressgateway \
```

Comment:
This can be done in the operator yaml as well

https://istio.io/latest/docs/reference/config/istio.operator.v1alpha1/#IstioComponentSetSpec